src/ens.ts

```diff
@@ -137,9 +144,9 @@ export default class Ens extends NamingService {
     const nodeHash = this.namehash(domain);
     const ownerPromise = this.owner(domain);
     const resolver = await this.getResolver(nodeHash);
-    if (!resolver || resolver === NullAddress) {
+    if (!resolver || resolver === NullAddress || resolver === NullAddressExtended) {
```

Comment:
Found out that the NullAddress returned by resolver has more '0' inside... Created NullAddressExrtended for now, looking for a better solution

Reply:
It is how it is. Looks like they can return an arbitrary string, I am worried that somewhere on the blockchain there can be another one with different length.